apps/gk_species_lbo.c

```diff
@@ -31,7 +31,7 @@ gk_species_lbo_init(struct gkyl_gyrokinetic_app *app, struct gk_species *s, stru
   if (s->info.collisions.normNu) {
     lbo->normNu = true;
     lbo->self_nu_fac = s->info.collisions.self_nu_fac;
-    double tpar_min = (s->info.mass/6.0)*s->grid.dx[cdim];
+    double tpar_min = (s->info.mass/6.0)*s->grid.dx[cdim]*s->grid.dx[cdim];
```

Comment:
@jRoeltgen I now realize tperp_min needs to be fixed too for the 1x1v case. If vdim=1, line 35 will lead to a memory error. Can you place it in a ternary like
```
tperp_min = vdim>1? (s->info.collisions.bmag_mid/3.0)*s->grid.dx[cdim+1] : tpar_min;
```
? thanks

Reply:
What is policy on vdim=1, then? I was under the impression it was being phased out in gk.